### ISSUE TITLE
Filter: Being able to define the color of the highlight

### ISSUE THREAD

**Issue by ididntknowicoulddothatXD** [ihavenoface/4chan-x#239]
Ever since I bought a 4chan Pass I highlight every ban evading tripfag I see, so I can report their threads and posts efficiently and almost instantly. As we know, the highlighting only allows the posts being highlighted in a reddish tone. I only highlight ban-evaders as of now. 
But I would also like to highlight tripturds who AREN'T ban evading with every post but are still breaking several rules at times, like avatarfagging. I divided my filter list in separate subjects but that really never satisfied me, as the obnoxious avatarfags aren't ban evading yet shared the same color.

TL;DR: Requesting the ability to choose various colors when highlighting.

**Comment by ihavenoface** [comment]
You can give a highlight a class and customize the color of that class with Custom CSS.

Do it like so:

Filter and add a class in the filter: `/!!annoyingPerson/i;highlight:someRetard;`

Apply css to the class: 

```
.someRetard > .reply {
  box-shadow: -5px 0px rgba(0, 128, 0, 0.5);
}
```

**Comment by ididntknowicoulddothatXD** [comment]
Thanks, didn't know you could do that.
